docs/content/getting-started/install.md

```diff
@@ -138,23 +140,144 @@ Check out:
+Next, we'll add some folders into your project to give you an idea of how the
+folder system works.
 
-For more information on configuring this feature, check out our guide at [Configuring Direnv](../guides/configuring_direnv.md)
+> **Tip:** It's often good practice to put the folder structure inside its own root folder. That way the folders will be grouped together and easy to distinguish from other folders. As an example, look at [NumTide treefmt](https://github.com/numtide/treefmt). 
 
+Let's start with a root folder to hold the other folders. We'll use "nix"
+as that's the standard one we've created. Open up your root flake.nix file
+and expand the outputs line so it takes up multiple lines, and then add 
+in the following prefix attribute:
 
-## Creating a root folder
+```nix
+  outputs = inputs:
+    inputs.blueprint {
+      inherit inputs;
+      prefix = "nix/";
+    };
+```
 
+Now create a `nix` folder at the root of your project alongside the flake.nix
+and devshell.nix files.
 
+Now you're ready to create some folders.
 
+First, remember that folders are detected automatically by Blueprint.
+That way, you can drop in 
+place pre-built flakes. For example, on another project, you might have
+built a flake that configures mysql. In that project you placed it in
+a folder called packages. You can then simply create a folder in your new
+project also called packages, and drop the mysql file in there, and you're
+good to go. No messing around with giant monolithic flake.nix file.
 
+Let's do something similar. Let's add some documentation to your app.
+Suppose we want to set up MkDocs with your project.
 
-## Adding a host
+> **Tip:** Remember, the whole point of Nix is to be able to set up reproducible
+environments. What that means is you don't need to install MkDocs globally. Instead,
+you can configure it directly in your project.
 
-TODO
+1. Under the `nix` folder, create another folder called `packages` (all lowercase).
+2. Then under `packages` create a folder called `docs`.
+3. Inside the `docs` folder, paste the following code into a file called `default.nix`:
 
-## Adding a package
+```nix
+{
+  pkgs,
+  perSystem,
+  ...
+}:
+pkgs.stdenvNoCC.mkDerivation {
+  name = "docs";
+
+  unpackPhase = ''
+    cp ${../../../mkdocs.yml} mkdocs.yaml
+    cp -r ${../../../docs} docs
+  '';
+
+  nativeBuildInputs = with pkgs.python3Packages; [
+    mike
+    mkdocs
+    mkdocs-material
+    mkdocs-awesome-pages-plugin
+  ];
+
+  buildPhase = ''
+    mkdocs build
+  '';
+
+  installPhase = ''
+    mv site $out
+  '';
+}
+```
+
+> Because Blueprint is present, this code will get located automatically. And notice
+how it can be reused; indeed for this example, we simply copied it over from the
```

Comment:
I think newslines break the quote here. Also probably intended a "Tip:" or something here?